lms/static/scripts/frontend_apps/config.ts

```diff
@@ -284,6 +285,9 @@ export type ConfigObject = {
 
   // Only present in "email-preferences" mode.
   emailPreferences?: EmailPreferences;
+
+  // Only present when in "analytics" mode.
+  users?: any[];
```

Comment:
We may want to start splitting this config per mode, using the `mode` prop as discriminator.